deploy/deploy.lib

```diff
@@ -352,6 +352,18 @@ configure_compilers() {
         cmd=${cmd/load/unload}
         echo "${cmd}"
         ${cmd}
+
+        cmd=${cmd/unload/show/}
+        full_module_path="$(${cmd}|&sed -ne '2{s/:$// p}')"
+        spec=${spec%%[+ ]*}             # truncate variants
+        spec=${spec/llvm/.clang}        # LLVM identifies as clang with some garbage in front
+        spec=${spec/20.0.2/19.1.2.254}  # Intel identifies as an older version
```

Comment:
I don't understand details here but noticed version hardcoding here...

Reply:
Yes… normally, Intel would just append a build number or so to the version, but for `intel@20`, `icc` actually quotes a version number starting with `19`!